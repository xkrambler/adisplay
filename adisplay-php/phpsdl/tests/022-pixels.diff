003+ Segmentation fault (core dumped)
004- Notice: SDL_Pixels::__construct(): Pitch set to 12 in %s%e022-pixels.php on line 8
005+ Termsig=11
005- object(SDL_Pixels)#%d (3) {
006-   ["pitch"]=>
007-   int(12)
008-   ["h"]=>
009-   int(10)
010-   ["count"]=>
011-   int(120)
012- }
013- string(17) "SDL_Pixels(12,10)"
014- = Get/Set
015- 
016- Notice: SDL_Pixels::GetByte(): Invalid position (15,15) in SDL_Pixels (12,10) in %s%e022-pixels.php on line 10
017- bool(false)
018- int(0)
019- int(1)
020- int(0)
021- int(255)
022- = Array
023- int(0)
024- bool(true)
025- bool(false)
026- int(127)
027- int(127)
028- int(0)
029- = Done