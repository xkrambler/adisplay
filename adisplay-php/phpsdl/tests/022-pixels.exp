= Create
Exception: Invalid size

Notice: SDL_Pixels::__construct(): Pitch set to 12 in %s%e022-pixels.php on line 8
object(SDL_Pixels)#%d (3) {
  ["pitch"]=>
  int(12)
  ["h"]=>
  int(10)
  ["count"]=>
  int(120)
}
string(17) "SDL_Pixels(12,10)"
= Get/Set

Notice: SDL_Pixels::GetByte(): Invalid position (15,15) in SDL_Pixels (12,10) in %s%e022-pixels.php on line 10
bool(false)
int(0)
int(1)
int(0)
int(255)
= Array
int(0)
bool(true)
bool(false)
int(127)
int(127)
int(0)
= Done