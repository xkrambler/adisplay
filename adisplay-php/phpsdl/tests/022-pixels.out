= Create
Exception: Invalid size
Segmentation fault (core dumped)

Termsig=11